frontend/src/views/DemandCapacityNotificationView.tsx

```diff
@@ -0,0 +1,110 @@
+import { Tab, TabPanel, Tabs, Table } from '@catena-x/portal-shared-components';
+import { Box, Button, Stack, Typography } from '@mui/material';
+import { getDemandAndCapacityNotification } from '@services/demand-capacity-notification';
+import { useEffect, useState } from 'react';
+import { Add } from '@mui/icons-material';
+import { DemandCapacityNotificationInformationModal } from '@features/notifications/components/NotificationInformationModal';
+import { DemandCapacityNotification } from '@models/types/data/demand-capacity-notification';
+import { EFFECTS } from '@models/constants/effects';
+import { LEADING_ROOT_CAUSE } from '@models/constants/leading-root-causes';
+import { STATUS } from '@models/constants/status';
+
+
+export const DemandCapacityNotificationView = () => {
+
+    const [selectedTab, setSelectedTab] = useState<number>(0);
+    const [demandCapacityNotification, setDemandCapacityNotification] = useState<DemandCapacityNotification[]>([]);
+    const [modalOpen, setModalOpen] = useState<boolean>(false);
+    const [selectedNotification, setSelectedNotification] = useState<DemandCapacityNotification | null>(null);
+
+    const tabs = ['Incoming', 'Outgoing'];
+
+    const fetchAndLogNotification = async () => {
+        try {
+            const result = await getDemandAndCapacityNotification(selectedTab === 0);
+            setDemandCapacityNotification(result);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    useEffect(() => {
+        fetchAndLogNotification();
+    }, [selectedTab]);
+
+    const TabPanelContent = ({ notifications }: { notifications: DemandCapacityNotification[] }) => {
+        return notifications.length !== 0 ?
+            <DemandCapacityNotificationTable onRowSelected={(notification) => {
+                setModalOpen(true);
+                setSelectedNotification(notification);
+            }} notifications={notifications} /> :
+            <Typography>No notifications available</Typography>
+    }
+
+    return (
+        <>
+            <Stack spacing={2} alignItems='center' width='100%' height='100%'>
+                <h1 className="text-3xl font-semibold text-gray-700 mb-10">Demand And Capacity Notifications</h1>
+                <Tabs value={selectedTab} onChange={(_, value: number) => setSelectedTab(value)}>
+                    {tabs.map((tab, index) => <Tab key={index} label={tab} />)}
+                </Tabs>
+                <Button variant="contained" onClick={() => {
+                    setSelectedNotification(null);
+                    setModalOpen(true)
+                }
+                }>
+                    <Add></Add> Send Notification
+                </Button>
```

Comment:
I would suggest aligning the tabs to the left and having the button aligned to the right on the same line.